packaging/server/build-backend.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -xe
+
+# Remove bindings crates that aren't needed for backend-server
+sed -i -e '\;oxidation/bindings;d' Cargo.toml
+
+# Remove the line that generate pyqt code in the build script since the backend don't need it.
+sed -i -e '\;misc/generate_pyqt.py;d' build.py
```

Comment:
This is too fragile, I'd rather introduce a `POETRY_PYQT_BUILD_STRATEGY=no_build` environ variable

Reply:
done